lighthouse-core/audits/load-fast-enough-for-pwa.js

```diff
@@ -41,7 +40,7 @@ class LoadFastEnough4Pwa extends Audit {
       category: 'PWA',
       name: 'load-fast-enough-for-pwa',
       description: 'Page load is fast enough on 3G',
-      helpText: 'Satisfied if the _Time To Interactive_ duration is shorter than _10 seconds_, as defined by the [PWA Baseline Checklist](https://developers.google.com/web/progressive-web-apps/checklist). Network throttling is required (specifically: RTT latencies >= 150 RTT are expected).',
```

Comment:
we'll need to decide if we want to essentially rename "FI" to "TTI" and bury the old one as version 0.1.

if we dont then we'll have to change this text.